src/index.js

```diff
@@ -1,6 +1,25 @@
 import './style.css';
-import { scoreList, loadList } from './load.js';
+import { addScore, displayScores } from './api.js';
 
-const ulContain = document.querySelector('#score_list');
+const form = document.querySelector('#form');
+const refresh = document.getElementById('refresh_btn');
+const yourname = document.querySelector('#yourname');
+const yourscore = document.querySelector('#yourscore');
```

Comment:
- [ ] The same thing is happening here. you are trying to select elements with id `yourname` & `yourscore` although in your index.html file there are no elements that have this id. please give your elements these ids so they can be found.
- [ ] [OPTIONAL] please give the `input` element for the score a `type="number"` because the score should ideally be a number not just any text.

Reply:
Hi @MrBrN197, thank you for a descriptive review. I have must be confused myself while I was pushing the files and merging the branches, sorry for this inconvenience.